lib/SILGen/SILGenStmt.cpp

```diff
@@ -1257,7 +1255,6 @@ void StmtEmitter::visitForEachStmt(ForEachStmt *S) {
 
     JumpDest loopDest = createJumpDest(S->getBody());
 
-    // Set the destinations for 'break' and 'continue'.
     JumpDest endDest = createJumpDest(S->getBody());
```

Comment:
Since we moved the comment, I think we can also get rid of the blank line between these two variables.